cmd/execute.cpp

```diff
@@ -16,96 +16,142 @@
 
 #include <CLI/CLI.hpp>
 #include <boost/endian/conversion.hpp>
-#include <gsl/gsl_util>
 #include <iostream>
 #include <limits>
+#include <regex>
 #include <stdexcept>
 
+#include <silkworm/db/chaindb.hpp>
+#include <silkworm/db/tables.hpp>
+#include <silkworm/db/util.hpp>
+
 #include "tg_api/silkworm_tg_api.h"
 
-static constexpr const char* kMigrationsTable{"migrations"};
-static constexpr const char* kDatabaseInfoTable{"DBINFO"};
-static constexpr const char* kSyncStageProgressTable{"SSP2"};
-static constexpr const char* kExecutionStage{"Execution"};
-static constexpr const char* kStorageModeReceipts{"smReceipts"};
+static constexpr const char* KExecutionStage_key{"Execution"};
+static constexpr const char* KStorageModeReceipts_key{"smReceipts"};
+
+std::optional<uint64_t> parse_size(const std::string& strsize) {
```

Comment:
Let's move parse_size into core silkworm to avoid code duplication between check_senders, dbtool, and execute.